module/VuFind/src/VuFind/Form/Handler/HandlerInterface.php

```diff
@@ -0,0 +1,57 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Interface HandlerInterface
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Moravian Library 2022.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Form
+ * @author   Josef Moravec <moravec@mzk.cz>
+ * @license  https://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+namespace VuFind\Form\Handler;
+
+/**
+ * Interface HandlerInterface
+ *
+ * @category VuFind
+ * @package  Form
+ * @author   Josef Moravec <moravec@mzk.cz>
+ * @license  https://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+interface HandlerInterface
+{
+    /**
+     * Gets data from submitted form and process them.
```

Comment:
Minor grammar issue: should be either "Gets ... and processes" or "Get ... and process" so that the clauses match. Whichever option you choose should be applied here and also to the two implementing classes.

Reply:
Fixed now